ext/data/schemas/dictionary-index-schema.json

```diff
@@ -47,6 +47,14 @@
             "type": "string",
             "description": "Attribution information for the dictionary data."
         },
+        "sourceLanguage": {
+            "type": "string",
+            "description": "Language of the terms in the dictionary."
+        },
+        "targetLanguage": {
+            "type": "string",
+            "description": "Main language of the definitions in the dictionary."
```

Comment:
What happens if there are multiple languages in the definition?

Reply:
(this is a thing with the words.hk dictionary going YUE -> YUE/EN)

Reply:
Yeah, I've seen some dictionaries like that. Even Jitendex technically has Japanese examples and whatnot in the definitions, which is why I wrote *Main* language. I guess we can see how this gets used and add an option for it being an array later on.